server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -36,6 +38,7 @@ export interface RegionPickerOption extends CloudLocationOption {
 export class OutlineRegionPicker extends LitElement {
   @property({type: Array}) options: RegionPickerOption[] = [];
   @property({type: Number}) selectedIndex = -1;
+  @property({type: Boolean}) metricsEnabled = false;
```

Comment:
I think we should we make it enabled by default?

@maddyhof @sbruens thoughts?

Reply:
Let's keep the consent model the same for now and discuss further. We can make that change separately.

It would also be kind of bad right now with the `if-messages` check as it would not allow users to opt-out until all messages were translated (though we're unlikely to launch in between this PR and localization coming in).